src/components/Tooltip/Tooltip.scss

```diff
@@ -0,0 +1,21 @@
+$bgColor: var(--color-gray_dark);
+
+.moonstone-tooltip_label {
+    z-index: 1;
+
+    width: inherit;
+    padding: var(--spacing-nano) var(--spacing-small);
+
+    font-size: 90%;
+    color: var(--color-white);
+    text-overflow: clip;
```

Comment:
If we use the `Typography` component we won't need to set `font-size` and `text-overflow`